R/themes.R

```diff
@@ -339,103 +339,70 @@ ThemeAestheticSelections <- R6::R6Class(
 #' @field addLine theme properties for `PlotConfiguration` objects as used in function `addLine()`
 #' @field addRibbon theme properties for `PlotConfiguration` objects as used in function `addRibbon()`
 #' @field addErrorbar theme properties for `PlotConfiguration` objects as used in function `addErrorbar()`
-#' @field plotPKRatio theme properties for `PlotConfiguration` objects as used in function `plotPKRatio()`
-#' @field plotDDIRatio theme properties for `PlotConfiguration` objects as used in function `plotDDIRatio()`
-#' @field plotTimeProfile theme properties for `PlotConfiguration` objects as used in function `plotTimeProfile()`
-#' @field plotObsVsPred theme properties for `PlotConfiguration` objects as used in function `plotObsVsPred()`
-#' @field plotBoxWhisker theme properties for `PlotConfiguration` objects as used in function `plotBoxWhisker()`
-#' @field plotTornado theme properties for `PlotConfiguration` objects as used in function `plotTornado()`
-#' @field plotHistogram theme properties for `PlotConfiguration` objects as used in function `plotHistogram()`
 #' @export
 ThemePlotConfigurations <- R6::R6Class(
   "ThemePlotConfigurations",
+  # This allows the R6 class to accept new fields
+  lock_objects = FALSE,
```

Comment:
interesting feature :)

Reply:
Totally.
I think it could be beneficial to apply it to other R6 objects such as workflows or task settings in RE (that would make user defined tasks or settings more flexible)